primitives/src/lib.rs

```diff
@@ -80,6 +80,21 @@ impl Default for CommitteeSeats {
     }
 }
 
+#[derive(Decode, Encode, TypeInfo, Eq, PartialEq)]
+pub struct EraValidators<AccountId> {
+    pub reserved: Vec<AccountId>,
+    pub non_reserved: Vec<AccountId>,
+}
+
+impl<AccountId> Default for EraValidators<AccountId> {
+    fn default() -> Self {
+        Self {
+            reserved: Vec::new(),
+            non_reserved: Vec::new(),
```

Comment:
With `#[derive(Default)]` it doesn't compile - do you know how to fix it?

Reply:
that's weird, because this works:
```rust
impl<AccountId> Default for EraValidators<AccountId> {
    fn default() -> Self {
        Self {
            reserved: Default::default(),
            non_reserved: Default::default(),
        }
    }
}
```
let's leave it